lib/wallet/payment_history_list_item.dart

```diff
@@ -27,6 +27,18 @@ class PaymentHistoryListItem extends StatelessWidget {
 
     String label;
     switch (data.type) {
+      case PaymentType.deposit:
```

Comment:
🔧 On the service model we used an extensions for that. I kinda like the extension approach better, as it better hides this ugly switch 😅

Reply:
feel free to create follow up for this :)